ams/server/src/main/java/com/netease/arctic/server/catalog/CatalogBuilder.java

```diff
@@ -55,14 +55,19 @@ public static ServerCatalog buildServerCatalog(CatalogMeta catalogMeta, Configur
         } else {
           throw new IllegalStateException("AMS catalog support iceberg/mixed-iceberg table only.");
         }
+      case CATALOG_TYPE_GLUE:
+        if (TableFormat.ICEBERG == tableFormat) {
```

Comment:
MixedIceberg could be created in glue too.

Reply:
As disscussed in #2007, Glue can only worked with S3 now, but Mixed Format is not fully supported on S3 yet, so I didn't support Mixed Format with Glue.

<img width="626" alt="image" src="https://github.com/NetEase/amoro/assets/103108928/c798cee9-d062-4a67-b214-73f923123b04">

Reply:
I support it, just like the custom catalog.